libraries/AP_AerobridgeTrustedFlight/AP_AerobridgeTrustedFlight.cpp

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
ensure that new cpp files have gone through astyle check

Reply:
done!